packages/jsii-pacmak/lib/dependency-graph.ts

```diff
@@ -58,6 +58,7 @@ export interface TraverseDependencyGraphHost {
 export interface PackageJson {
   readonly dependencies?: { readonly [name: string]: string };
   readonly peerDependencies?: { readonly [name: string]: string };
+  readonly bundledDependencies?: string[];
```

Comment:
This can be named `bundleDependencies` or `bundledDependencies` (they are synonyms). Probably should account for that... As we don't mandate people use one or the other.